modules/accredited_representative_portal/db/migrate/20250121164657_prepare_ar_pos_holder_data.rb

```diff
@@ -0,0 +1,10 @@
+class PrepareArPosHolderData < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
```

Comment:
Can you remove this line? We don't want to disable this if we don't have to. And we only have to when adding or removing indexes.